### ISSUE TITLE
Offset overlay image during inpainting

### ISSUE THREAD

**Issue by dougbtv** [AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin#351]
Using version v1.3.0
Using [vladautomatic/sd-next @ latest](https://github.com/vladmandic/automatic/commit/417ef540039e2d89a02cefb4fea2d767a5ba1380)

Whenever I inpaint with a selection, I get a weird offsetting of the resulting image from the inpaint.

I've attached a screenshot as it's difficult to explain otherwise.

![inpaint-offset-problem-sd-photoshop](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/5472189/9cccc382-73b7-43cf-a7ca-210519690b4a)

**Comment by huchenlei** [comment]
I can confirm that issue is caused by a1111 side, not the extension. I have the same issue in my photopea plugin. 

I worked around it by cropping the result image to mask selection range.
https://github.com/huchenlei/stable-diffusion-ps-pea/issues/1

**Comment by dougbtv** [comment]
Thanks @huchenlei !! I also just confirmed that it's not present in  a1111 v1.5.1 as far as I tested.

I'll file an issue on vladmandic 👍

**Comment by huchenlei** [comment]
I am not sure a1111 fixed this recently but I still had issue in 1.5.1 a1111. The issue is more obvious if your inpaint area is close to the border of an image.

**Comment by dougbtv** [comment]
Yeah, somehow I can't reproduce with 1.5.1, oddly enough. I do think it's not the photoshop extension side though after running this through test. 

I can get a mismatched edge if I mask at the border of a selection, but it's not offset elsewhere, so I don't think it's the same thing (I think it's going to always mismatch when done this way) example attached.

Also big thanks for the response and confirming it's on the webui side.

![no-reproduction](https://github.com/AbdullahAlfaraj/Auto-Photoshop-StableDiffusion-Plugin/assets/5472189/af0989bf-d87f-437f-93f4-113ed7c4c845)

**Comment by cdmusic2019** [comment]
You can try using the "inpainting" model.
